src/setupProxy.js

```diff
@@ -3,7 +3,7 @@ const { createProxyMiddleware } = require("http-proxy-middleware");
 module.exports = function (app) {
   app.use(
     createProxyMiddleware("/api", {
-      target: "https://dev.internalapi.biip.lt/uetk",
+      target: "https://dev-internalapi.biip.lt/uetk",
```

Comment:
butu geriau cia imti is `.env`

Reply:
Sitas projektas i vite nepermigruotas, tai gal geriau atskiru pull requestu permigruosiu ir tada dar patvarkysiu env